src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/coreperson/CorePersonMappingResource.kt

```diff
@@ -91,7 +91,7 @@ class CorePersonMappingResource(private val service: CorePersonService) {
       )
     }
 
-  @GetMapping("/person/migration-id/{migrationId}")
+  @GetMapping("/migration-id/{migrationId}")
```

Comment:
migration endpoint no longer has /person in path (not needed - highlighted in previous PR)